android/app/src/main/java/network/mysterium/service/core/NodeRepository.kt

```diff
@@ -51,8 +47,29 @@ class Status(
         val serviceType: String
 )
 
+class Identity(
+        val address: String,
+        val channelAddress: String,
+        val registrationStatus: String
+)
+
+class IdentityRegistrationFees(
+        val fee: Long
+)
+
+class Balance(val value: Long)
+
+// Wrapper around Go mobile node library bindings. It should not change any result
+// returned from internal mobile node and instead all mappings should happen in
+// ViewModels.
 class NodeRepository(private val deferredNode: DeferredNode) {
 
+    // Get available proposals for mobile. Internally on Go side
+    // proposals are fetched once and cached but it is possible to refresh cache by
+    // passing refresh flag.
+    //
+    // Note that this method need to deserialize JSON byte array since Go Mobile
+    // does not support passing complex slices via it's bridge.
     suspend fun getProposals(refresh: Boolean): List<ProposalItem> {
```

Comment:
In this case method accepts argument. If I rename it will look like a getter property. `nodeRepository.proposals(true)` will look strange. For example kotlin allows to access java methods getSomething automatically as something, but only if it doesn't contain any argument.

Reply:
Use named arguments: `repository.proposals(refresh: true)`.

Reply:
Kotlin and other newer languages support accessing properties directly for the very same reason: prefix does not add any value.